components/Side/Bar.vue

```diff
@@ -1,4 +1,216 @@
+<script>
+export default {
+  computed: {
+    $style() {
+      return {
+        link: "flex w-[250px] h-[50px] items-center rounded-[8px] hover:bg-base hover:text-white mt-[10px]",
+        text: "ml-[14px] text-[#333] group-hover:text-[#fff]",
+      };
+    },
+  },
+};
+</script>
 <template>
-    <div>
+  <div class="h-screen w-[300px] flex flex-col items-center bg-white">
+    <h1 class="text-[26px] text-[#333] p-[20px]">Dashboard</h1>
+    <div class="mt-[30px]">
+      <div :class="$style.link" class="group duration-500">
```

Comment:
`<div :class="$style.link" class="group duration-500"> ... </div>` should be wrapped with:
`<NuxtLink to="/"> ... </NuxtLink>` (where "/" is a path to a page)
and declared as reusable component with example properties: svg, text, path etc.
SVG's should be imported from dedicated file (maybe assests/SVG/SvgName ?)

Components can be created based on declared objects array with `v-for` https://vuejs.org/api/built-in-directives.html#v-for